plonky2x/src/frontend/vars/variable.rs

```diff
@@ -107,3 +109,142 @@ impl<L: PlonkParameters<D>, const D: usize> One<L, D> for Variable {
         Variable(builder.api.one())
     }
 }
+
+impl<L: PlonkParameters<D>, const D: usize> LessThan<L, D> for Variable {
+    type Output = BoolVariable;
+
+    fn lt(self, rhs: Variable, builder: &mut CircuitBuilder<L, D>) -> Self::Output {
+        // FIXME: Check for overflows and edge cases
+        const NUM_LIMBS: usize = 64;
+        let max = builder.constant(L::Field::from_canonical_u64(1 << (NUM_LIMBS - 1)));
+        let _one: Variable = builder.constant(L::Field::from_canonical_u64(1));
```

Comment:
nit: I think you should just do `builder.one::<Variable>()` here

Reply:
fixed in [96d77c3](https://github.com/succinctlabs/succinctx/pull/152/commits/96d77c370e478a2372630da59b9cc996b23e4a0a)